.github/scripts/CheckIssueContent.ps1

```diff
@@ -0,0 +1,10 @@
+﻿#Params
+param([string]$issueContent="")
+
+#Processing
+function Check-For-Missing-Information([string]$issueContent){
+    return $issueContent -like '*(Fill in here)*(Fill in here)*'
```

Comment:
Can you put some comment to explain in the future how to modify this piece?

Reply:
Sure, I'll include it in the next update